### ISSUE TITLE
py.typed file is missing 

### ISSUE THREAD

**Issue by anentropic** [launchdarkly/python-server-sdk#171]
issue https://github.com/launchdarkly/python-server-sdk/issues/165 was closed via https://github.com/launchdarkly/python-server-sdk/pull/166

unfortunately that wasn't a complete fix, as the `py.typed` file that was added to the repo was not included in the source distribution sent to pypi: https://files.pythonhosted.org/packages/55/0a/7bf239d8550745fdde6d7a26788c1e9aac2a1628586e22cac8a1d8d61bae/launchdarkly-server-sdk-7.4.1.tar.gz

...so mypy still gives:
```
error: Skipping analyzing "ldclient": module is installed, but missing library stubs or py.typed marker
```

**Comment by keelerm84** [comment]
Acknowledging here that we have seen this issue and discussion will continue on the PR.

**Comment by keelerm84** [comment]
Fixed in 7.5.0
